crates/uv-dispatch/src/lib.rs

```diff
@@ -397,6 +397,67 @@ impl<'a> BuildContext for BuildDispatch<'a> {
         .await?;
         Ok(builder)
     }
+
+    async fn direct_build<'data>(
+        &'data self,
+        source: &'data Path,
+        subdirectory: Option<&'data Path>,
+        output_dir: &'data Path,
+        build_kind: BuildKind,
+        version_id: Option<String>,
```

Comment:
Should this be a `Option<&str>`?

Reply:
I see. It would be kinda annoying, but doable. I don't feel strongly.

Reply:
It's more converting back and forth now, but it's also move consistent with the rest of the codebase.